main.go

```diff
@@ -48,6 +51,13 @@ func main() {
 				Usage:       "Show what would be done, but don't actually publish",
 				Destination: &dryRun,
 			},
+			&commandLine.StringFlag{
```

Comment:
[MultiStringFlag](https://pkg.go.dev/github.com/urfave/cli/v2@v2.10.2#MultiStringFlag) would probably be a little easier to manage

Reply:
Updated.